src/PrestaShopBundle/Resources/views/Admin/Improve/International/Localization/Blocks/advanced_configuration.html.twig

```diff
@@ -32,18 +32,22 @@
       <i class="material-icons">settings</i> {{ 'Advanced'|trans({}, 'Admin.Global') }}
     </h3>
 
-    <div class="card-block">
+    <div class="card-block row">
       <div class="card-text">
-        <div class="form-group">
+        <div class="form-group row">
           {{ ps.label_with_help(('Language identifier'|trans), ('The ISO 639-1 identifier for the language of the country where your web server is located (en, fr, sp, ru, pl, nl, etc.).'|trans({}, 'Admin.International.Help'))) }}
```

Comment:
Actually, there is like a very silly mistake in this string... in the enumeration part, I guess _sp_ stands for the Spanish language, but this 639-1 code does not exist, it should be _es_.